### PR TITLE
completed javascript-if

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH >= 0 && pH < 7) {
+    return 'acid';
+  } else if (pH <= 14 && pH >= 8) {
+    return 'base';
```
Comment (TimelikeClosure) [inline javascript-if/main.js:61]: pH levels don't need to be whole numbers. For example, human blood is slightly basic at a pH of 7.4. Your `categorizeAcidity` function should be able to categorize that properly:

---

image

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  } else {
+    return 'invalid pH level';
+  }
+}
+
+function introduceWarnerBro(name) {
+  if (name === 'yakko' || name === 'wakko') {
+    return 'We\'re the warner brothers!';
+  } else if (name === 'dots') {
+    return 'I\'m cute~';
```
Comment (TimelikeClosure) [inline javascript-if/main.js:71]: It's `"dot"`, not `"dots"`:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 2 above)
Comment (m-Cho1) [reply javascript-if/main.js]: image
edited

**Comment by m-Cho1** [comment]
image

File: javascript-if/main.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH >= 0 && pH < 7.0) {
+    return 'acid';
+  } else if (pH <= 14 && pH >= 7.1) {
+    return 'base';
+  } else {
+    return 'invalid pH level';
+  }
```
Comment (TimelikeClosure) [inline javascript-if/main.js:64]: Still not quite right: a pH level of `7.0001` is still a `"base"`.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
image
edited

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**